src/main/java/de/bonndan/nivio/input/StartupListener.java

```diff
@@ -57,7 +58,19 @@ public void onApplicationEvent(final ApplicationReadyEvent event) {
         List<URL> landscapeDescriptionLocations = new ArrayList<>();
         try {
```

Comment:
Please consider to move the try-catch inside the loop over seed.getLocations(), so we're able to report single broken URLs and keep the rest.